src/components/modals/SingleInputModal.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+// $FlowFixMe
```

Comment:
Same comment as in here: https://github.com/EdgeApp/edge-react-gui/pull/2625#discussion_r636652888

Reply:
because branch about `$FlowFixMe` already merged, I use `src/utils/hooks` now